cmd/kubelet/app/server.go

```diff
@@ -357,6 +357,13 @@ func loadConfigFile(name string) (*kubeletconfiginternal.KubeletConfiguration, e
 	if err != nil {
 		return nil, fmt.Errorf(errFmt, name, err)
 	}
+
+	// EvictionHard may be nil if it was not set in kubelet's config file.
```

Comment:
update the godoc on the `EvictionHard` field in `types.go` to indicate the `nodefs.inodesFree` default is linux-only